main.scss

```diff
@@ -37,21 +37,13 @@ $o-expander-is-silent: true !default;
 
 	i {
 		@include oFtIconsBaseIconStyles();
```

Comment:
Actually, it's the default, so this shouldn't be necessary. You can probably set it to false in the expanded icon, and in that case, you wouldn't need to set the $container-height parameter